elliottlib/bzutil.py

```diff
@@ -1063,8 +1063,17 @@ def is_first_fix_any(bugtracker, flaw_bug, current_target_release):
         return True
 
     # filter tracker bugs by OCP product
-    tracker_bugs = [b for b in bugtracker.get_bugs(tracker_ids)
-                    if b.product == constants.BUGZILLA_PRODUCT_OCP and b.is_tracker_bug()]
+    tracker_bugs = []
+    for tracker_id in tracker_ids:
+        try:
+            b = bugtracker.get_bug(tracker_id)
+        except Exception as e:
+            # first-fix tracker bug might be not visible but not break here
+            logger.warn(e)
```

Comment:
I think we could use a more explanatory warning message to grep the logs for when needed

Reply:
as previous, I can't distinguish the unauthorized error, so print all message here

Reply:
I think this can still be improved: https://github.com/openshift/elliott/pull/366#discussion_r921000858